internal/internal.go

```diff
@@ -18,6 +17,8 @@ import (
 	"sync"
 	"time"
 
+	"github.com/open-policy-agent/opa/topdown"
```

Comment:
I'd put it into the last lump to the other `github.com/open-policy-agent/opa` deps, but this is fine. ;)